packages/dropdown/src/react/selected.tsx

```diff
@@ -1,59 +1,43 @@
-import { ValueOf } from '@pluralsight/ps-design-system-util'
-import glamorDefault, * as glamorExports from 'glamor'
+import { ValueOf, classNames } from '@pluralsight/ps-design-system-util'
 import React from 'react'
 
-import stylesheet from '../css/index'
+import '../css/index.css'
 import { ItemData } from '../js/index'
 import * as vars from '../vars/index'
 
-const glamor = glamorDefault || glamorExports
-
-const styles = {
-  buttonSizer: () => glamor.css(stylesheet['.psds-dropdown__button-sizer']),
-  placeholder: ({
-    appearance,
-    selectedItem,
-    size
-  }: Pick<DropdownSelectedProps, 'appearance' | 'selectedItem' | 'size'>) => {
-    const label = 'psds-dropdown__placeholder'
-    const isSmall = size === vars.sizes.small
-    const placeholderColor =
-      appearance === vars.appearances.subtle
-        ? glamor.css(
-            stylesheet[
-              `.psds-dropdown__field--appearance-${vars.appearances.subtle}.psds-dropdown__placeholder--color`
-            ]
-          )
-        : glamor.css(stylesheet['.psds-dropdown__placeholder--color'])
-    return glamor.compose(
-      glamor.css(stylesheet[`.${label}`]),
-      isSmall && glamor.css(stylesheet[`.${label}.psds-dropdown--small`]),
```

Comment:
This is missing below

Reply:
Non-issue. See https://github.com/pluralsight/design-system/pull/1881#discussion_r664665330